### ISSUE TITLE
@angular/forms peer dependency points to specific version

### ISSUE THREAD

**Issue by ln-e** [DesertFoxNV/ngx-autosize-input#54]
[Published version](https://www.npmjs.com/package/ngx-autosize-input/v/15.0.0?activeTab=explore) 15.0.0 lock `@angular/forms` to `15.0.4`. Seems it should points to range like other peer deps

```
"peerDependencies": {
    "@angular/common": "^15.0.0",
    "@angular/core": "^15.0.0",
    "@angular/forms": "15.0.4",
    "rxjs": "7.5.7"
  },
```

**Comment by joshuawwright** [comment]
@ln-e Thanks this is a great suggestion. I will try to make this change this week.

**Comment by joshuawwright** [comment]
@ln-e this has been fixed in 15.0.1. Thanks for bringing this to my attention.
